File: backend/Projetos/KeySystems.ERP.Gateway/src/KeySystems.ERP.Core/DomainModel/Tenant.cs
using System;

namespace KeySystems.ERP.Core.DomainModel
{
    public class Tenant
    {
        public Tenant(int id, string nome)
        {
            this.Id = id;
            this.Nome = nome;
        }

        public int Id { get; set; }
        public string Nome { get; set; }
    }
}

File: frontend/src/app/shared/tenant/tenant.ts


export class Tenant {
    
    private _nome : string;
    public get nome() : string {
        return this._nome;
    }
    public set nome(v : string) {
        this._nome = v;
    }
    
    private _id : string;
    public get id() : string {
        return this._id;
    }
    public set id(v : string) {
        this._id = v;
    }
}